assets/js/app.js

```
@@ -1,264 +1,533 @@
+}
+
+function updateActiveSection(sectionId) {
+  state.activeSection = sectionId;
+
+  // Update navigation active states
+  document.querySelectorAll(".nav-link").forEach((link) => {
+    link.classList.remove("active");
   });
+
+  const activeLink = document.querySelector(`[data-section="${sectionId}"]`);
+  if (activeLink && activeLink.classList.contains("nav-link")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)